crates/uv-install-wheel/src/wheel.rs

```diff
@@ -186,6 +187,25 @@ pub(crate) fn write_script_entrypoints(
     is_gui: bool,
 ) -> Result<(), Error> {
     for entrypoint in entrypoints {
+        let warn_names = ["activate", "activate_this.py"];
```

Comment:
Why not just check if `entrypoint.name` starts with "activate"?

Reply:
It avoids detecting other scripts that are called something with `activate`, the warnings are intentionally scoped tightly to avoid false positives.